index.html

```diff
@@ -9,20 +9,30 @@
         <link rel="stylesheet" href="./style.css">
     </head>
     <body>
+
         <header class="appbar header">
-            <div class = "toolbar">
-                <div class="mylogo">
+            <div class = "toolbar mobile">
+                <div class="mylogo-mobile mobile">
                     <a href="">My Logo</a>
                 </div>
-                <nav class="menu-container">
-                    <button class="button-burger" href="">
+                <nav class="menu-container mobile">
+                    <button class="button-burger mobile" href="">
                         <div class="burger"></div>
                         <div class="burger"></div>
                         <div class="burger"></div>
                     </button>
                 </nav>
             </div>
+            <nav class = "toolbar-d desktop">
+                <div class="mylogo-desktop desktop"><a href="">My Logo</a></div>
```

Comment:
- [ ] Kindly center your content on the desktop version
- [ ] Also remember to fix your social icons. it is not showing
<img width="1433" alt="Screenshot 2023-06-16 at 16 31 17" src="https://github.com/AliRezaBmeDu/Portfolio-Website/assets/81444338/523f0ec3-df11-4573-b491-df9cf53333f6">

Reply:
Please check my screenshot of the top page. Social icons were showing on my screen.
![socialIcon_issue](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/34375633/196be850-46d4-4cbb-a614-04c16d519300)